packages/cli/lib/project-generator.js

```diff
@@ -177,11 +194,11 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       },
     ];
     return this.prompt(prompts).then(props => {
-      const settings = props.settings || choices.map(c => c.name);
+      const settings = props.settings || choices.map(c => `Enable ${c.key}`);
```

Comment:
I find this suspicious. Here is what the docs is saying:

https://www.npmjs.com/package/inquirer#question
> `choices`: `(Array|Function)` Choices array or a function returning a choices array. If defined as a function, the first parameter will be the current inquirer session answers. Array values can be simple strings, or objects containing a `name` (to display in list), a `value` (to save in the answers hash) and a `short` (to display after selection) properties.

I think we should change the way we are building choices as follows:

 - Set `value` to `f.name` (e.g. value = `tslint`)
 - Keep `name` as it is now
 - Add `short` set to `Enable ${f.name}`

With those changes in place, I believe that `props.settings` will use the `f.name` as the key and no string manipulation (`Enable ${c.key}`) will be needed in this anonymous function.

What do you think?

Reply:
I think that's a better approach because it's clearer. I'll change it to this one. Thank you!

Reply:
On second thought, should we keep `value` to stay the same as it is now? It's currently a boolean to determine whether or not to enable a setting. We can use `short` in ` const settings = props.settings || choices.map(c => c.short);` as it's already `Enable ${c.key}`, but what do you think?